uaclient/contract.py

```diff
@@ -391,14 +391,22 @@ def _create_attach_forbidden_message(
 
         if reason == "no-longer-effective":
             date = info["time"].strftime(ATTACH_FAIL_DATE_FORMAT)
+            additional_info = {
+                "contract_expiry_date": info["time"].strftime("%m-%d-%Y")
```

Comment:
Thanks I think the non-nested is simpler :)

We also have the contract_id in the response here, should we include that as well?